src/modules/sync/components/sync-dialog.tsx

```diff
@@ -149,6 +154,20 @@ export function SyncDialog( {
 	const { fetchChildren, rewindId, isLoadingRewindId, isErrorRewindId, isLoadingLocalFileTree } =
 		useDynamicTreeState( type, localSite.id, remoteSite.id, setTreeState );
 
+	const [ wpVersion ] = useGetWpVersion( localSite );
+	const minimumWordPressVersion = useRootSelector( selectMinimumWordPressVersion );
+	const { data: wpVersions = [] } = useGetWordPressVersions( {
+		minimumVersion: minimumWordPressVersion,
+	} );
+	const latestWpVersion = getLatestStableWpVersion( wpVersions );
```

Comment:
Maybe I am missing something, but what if we did something like this?

```suggestion
	const { data: wpVersions = [] } = useGetWordPressVersions( { minimumVersion: '' } );
	const latestWpVersion = getLatestStableWpVersion( wpVersions );
```

Reply:
Good catch! Adjusted in https://github.com/Automattic/studio/pull/2136/commits/aa1cac95812c157666e324c45a489293cbd20950